src/blog/2024/06/using-timescaledb-with-node-red.md

```diff
@@ -0,0 +1,293 @@
+---
+title: Using TimescaleDB with Node-RED.
+subtitle: Guide to integrating TimescaleDB into Node-RED for Storing, Updating, and Querying Data.
+description: Learn how to integrate TimescaleDB with Node-RED for storing and managing time-series data efficiently.
+date: 2024-06-03
+authors: ["sumit-shinde"]
+image: 
+tags:
+   - posts
+   - node-red
+   - timescaledb
+   - postgresql
+   - time-series database
+---
+
+In the context of IoT and IIoT applications, time series databases are essential for storing data based on timestamps. While InfluxDB has been a popular choice for a long time, another time series database, TimescaleDB, is gaining popularity. This article will cover how to use TimescaleDB with Node-RED, when to choose it over InfluxDB, and important operations and queries needed when building IoT applications.
+
+If you prefer video tutorials, a few months ago, Gray, OT Data & Community Strategist at Flowfuse, conducted a live session on TimescaleDB. You can watch it [here](https://www.youtube.com/watch?v=MD1U6LDqJ1c).
+
+<!--more-->
+
+## What is TimeScaleDB
+
+TimescaleDB is a time-series database built on PostgreSQL for handling large time-stamped data efficiently. It's optimized for time-series data, which offers automatic partitioning, optimized indexing, compression, and advanced analytics like continuous aggregates. Unlike PostgreSQL, it focuses specifically on time-series data management with distributed hypertable architecture.
+
+## TimescaleDB Vs InfluxDB
```

Comment:
While I like the comparison between the two, I am curious if we want to get into the game of saying which is better or worse. Even though that isn't what you are saying, it will be interpreted that way by some.

I will defer to @ZJvandeWeg for guidance.

Reply:
Should we remove the comparison section?